website/blog/2020-11-10-lithium.md

```diff
@@ -0,0 +1,507 @@
+---
+author: Tomasz Godzik
+title: Metals v0.9.5 - Lithium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.9.5, which brings about some of
+the long awaited features such as implicit decorations and the organize imports
+code action. We also greatly simplified connecting to the sbt BSP server and
+added support for Scala 3's first milestone `3.0.0-M1`.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">178</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">66</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">9</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">33</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">5</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/28?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Organize imports code action.
+- Show implicit parameters and inferred type decorations.
+- Improved sbt BSP support.
+- Support for Scala 3.0.0-M1.
+- Remote debugging.
+- Environment variables in run/debug.
+
+## Organize imports code action
+
+One of the most requested features in the Metals repository, current list can be
+found [here](https://github.com/scalameta/metals/issues/707), was the ability to
+automatically organize imports. We are happy to announce, that thanks to
+[mlachkar's](https://github.com/mlachkar) amazing contributions to both
+[Scalafix](https://github.com/scalacenter/scalafix) and Metals, this new feature
+is now available via an `Organize imports` code action.
+
+![imports](https://i.imgur.com/8YBdjjC.gif)
+
+Depending on the editor this code action can be invoked differently, please
+consult your specific editor's documentation. For example in Visual Studio Code
+'organize imports' can be invoked from command console, shortcut, or from the
+dropdown menu when right clicking inside a file.
+
+The organize imports code action is enabled using
+[Scalafix](https://github.com/scalacenter/scalafix) and specifically the awesome
+organize imports rule created by [liancheng](https://github.com/liancheng).
+
+The rule can be used to automatically sort imports in a file by the Ascii order,
+which is the default setting, or use the user specific configuration defined
+using scalafix configuration file. This file can be either `.scalafix.conf` in
+the current workspace or an absolute file specified in
```

Comment:
```suggestion
the current workspace or an absolute file specified in the
```

Reply:
Ach, sorry! Fixed! I will start on the release in the afternoon.